### DISCUSSION TITLE
Extract the force and moment component at each grid of the surface marker

### DISCUSSION THREAD

**Discussion by Rashmikant15** [su2code/SU2#2385] [Q&A]
Dear SU2 community,

I have done aerodynamic analysis using SU2, and I want to extract the force (Fx, Fy, Fz) and moment components (Mx, My, Mz) at each grid of the surface marker in a specified order. Ex. from the root chord (leading-edge to trailing-edge) to the tip chord for an aircraft wing. 

Thank you so much for the help.

**Comment by bigfooted** [comment]
I do this in paraview. I have an example for the local and total forces of the unsteady flow over an unsteady cylinder here:
https://github.com/bigfooted/su2cases/tree/master
https://github.com/bigfooted/su2cases/tree/master/validation/unsteady_inc_cylinder_farfield

![image](https://github.com/user-attachments/assets/9200fb74-c5c9-40d2-a8bf-d85b42233e63)
